extensions/adobe/experience/campaign/journeyaiscores.json

```diff
@@ -0,0 +1,51 @@
+{
+  "meta:license": [
+    "Copyright 2018 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/experience/campaign/journeyaiscores",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Journey AI Scores",
+  "type": "object",
+  "description": "Journey AI scores for one channel",
+  "definitions": {
+    "journeyaiscores": {
+      "properties": {
+        "xdm:fatigue": {
+          "title": "Fatigue",
+          "description": "Fatigue score for the profile",
+          "$ref": "https://ns.adobe.com/experience/campaign/journeyaifatigue"
+        },
+        "xdm:openScoresBySendHourForWeek": {
```

Comment:
Will customers understand this, because I do not.

Reply:
We just wanted to add some context about the scores ( that they for a week, 24x7 scores ). I think title and description can be used for that, and I will rename the fields as `openScores` and `clickScores`